extensions/applicationinsights-clickanalytics-js/package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "@microsoft/applicationinsights-clickanalytics-js",
+  "version": "2.5.9",
+  "description": "Microsoft Application Insights Click Analytics extension",
+  "main": "dist/applicationinsights-clickanalytics-js.js",
+  "module": "dist-esm/applicationinsights-clickanalytics-js.js",
+  "types": "types/applicationinsights-clickanalytics-js.d.ts",
+  "sideEffects": false,
+  "scripts": {
+    "build": "npm run build:esm && npm run build:browser",
+    "build:esm": "grunt clickanalytics",
+    "build:browser": "rollup -c",
+    "test": "grunt clickanalyticstests",
+    "lint": "tslint -p tsconfig.json"
+  },
+  "devDependencies": {
+    "@microsoft/applicationinsights-rollup-es3": "1.1.3",
+    "@microsoft/applicationinsights-properties-js": "2.5.8",
+    "typescript": "2.5.3",
+    "globby": "^11.0.0",
+    "rollup-plugin-node-resolve": "^3.4.0",
+    "rollup-plugin-replace": "^2.1.0",
+    "rollup-plugin-uglify": "^6.0.0",
+    "grunt": "1.0.1",
+    "grunt-contrib-qunit": "^3.1.0",
+    "grunt-contrib-uglify": "3.1.0",
+    "grunt-ts": "^6.0.0-beta.15",
+    "rollup": "^0.66.0",
+    "tslint": "^5.19.0",
+    "tslint-config-prettier": "^1.18.0"
+  },
+  "dependencies": {
+    "@microsoft/dynamicproto-js": "^1.0.0",
+    "@microsoft/applicationinsights-shims": "1.0.2",
+    "@microsoft/applicationinsights-core-js": "2.5.8",
```

Comment:
nit: These should be 2.5.9 -- but I will update to 2.5.10 as part of the release process